content/automate/ManageIQ/Cloud/Orchestration/Reconfiguration/StateMachines/Methods.class/__methods__/postreconfigure.rb

```diff
@@ -1,19 +1,42 @@
 #
 # Description: This method examines the orchestration stack reconfigured
 #
-def dump_stack_outputs(stack)
-  $evm.log("info", "Outputs from updated stack #{stack.name}")
-  stack.outputs.each do |output|
-    $evm.log("info", "Key #{output.key}, value #{output.value}")
-  end
-end
+module ManageIQ
+  module Automate
+    module Cloud
+      module Orchestration
+        module Reconfiguration
+          module StateMachines
+            class PostReconfigure
+              def initialize(handle = $evm)
+                @handle = handle
+              end
 
-$evm.log("info", "Starting Orchestration Post-Reconfiguration")
+              def main
+                @handle.log("info", "Starting Orchestration Post-Reconfiguration")
 
-service = $evm.root["service_reconfigure_task"].source
-stack = service.orchestration_stack
+                service = @handle.root["service_reconfigure_task"].source
```

Comment:
@pkomanek 
We have to check if the @handle.root['service_reconfigure_task'] has a valid value.
Can we use a pattern like this
https://github.com/ManageIQ/manageiq-content/blob/master/content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/execute.rb#L36
and delegate everything to the service method